README.md

```diff
@@ -78,6 +80,50 @@ let ``Example of automatically updating`` () =
     }
 ```
 
+## Bulk update of snapshots
+
+*Warning*: when doing this, you should probably make sure your test fixture is `[<NonParallelizable>]` (or the equivalent in your test runner of choice).
+Otherwise, the global state used by this mechanism may interfere with other fixtures.
```

Comment:
Refine this: the tests can run in parallel (we should make the add method lock though), it's just that fixtures can't.